packages/manager/apps/telecom/src/app/app.less

```diff
@@ -110,6 +110,7 @@
 @import './telecom/pack/xdsl/access/deconsolidation/pack-xdsl-access-deconsolidation.less';
 @import './telecom/pack/xdsl/access/linediagnostic/pack-xdsl-access-linediagnostic.less';
 @import './telecom/pack/xdsl/access/pack-xdsl-access.less';
+@import './telecom/pack/xdsl/meetings/xdsl-meetings.less';
```

Comment:
Do we really need to create some more custom styles ?

Reply:
I'm on it. Just need some test case to finalize